app/controllers/uploads_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
   def new
     @upload = new_upload(params[:csv_type])
-    return if @upload.csv_type_check?
+    return if @upload.csv_type.blank? || @upload.csv_type_check?
```

Comment:
The original logic should be left in place if we agree to remove the "Upload File" button from this view, since the generic upload function was removed from the dashboard".